projects/mc_interface/src/main.c

```diff
@@ -83,10 +98,14 @@ int main(void) {
     .can_msg_id = SYSTEM_CAN_MESSAGE_MOTOR_RELAY,
     .left_relay = MC_CFG_RELAY_LEFT,
     .right_relay = MC_CFG_RELAY_RIGHT,
+    .left_sense = MC_CFG_SENSE_LEFT,
+    .right_sense = MC_CFG_SENSE_RIGHT,
     .delay_ms = MC_CFG_RELAY_DELAY_MS,
   };
 
   sequenced_relay_init(&s_relay_storage, &relay_settings);
+  soft_timer_start_millis(MC_CFG_PRECHARGE_CHECK_INTERVAL_MS, prv_check_precharge_relays,
```

Comment:
This should occur inside of the sequenced_relay_callback the causes precharge relays to close not in main.